src/stk/molecular/topology_graphs/host_guest/complex.py

```diff
@@ -4,8 +4,17 @@
 
 """
 
+from typing import Dict, Optional, Tuple, Iterable
```

Comment:
Use 
```python
from __future__ import annotations
```

and then you can use `tuple, dict` for type hints instead of importing `typing.Tuple` and `typing.Dict`